.github/workflows/replicate.yml

```diff
@@ -0,0 +1,30 @@
+name: Replicate
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: 50 4 * * *
+
+jobs:
+  replicate:
+    # TODO: Change this to website when merging
+    if: github.repository == 'exercism/v3-website'
+
+    name: Replicate to exercism/website-deployer
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
This is required to checkout the entire history of the branch and not just the latest commit